mumblepahelper.cpp

```diff
@@ -45,6 +46,18 @@ MumblePAHelper::~MumblePAHelper() {
 		delete plugins;
 }
 
+// We use an event filter to prevent the status bar message
+// from being cleared when moving the cursor over a menu option.
+// This happens because the menu sends a signal to the status bar
+// containing the option's tip, which is empty.
+bool MumblePAHelper::eventFilter(QObject* object, QEvent* event)
```

Comment:
This doesn't follow Mumble coding style: Curlys don't go on a new line. Pointer stars don't hug the type.